core/src/main/java/com/klaytn/caver/methods/response/Peers.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2021 The caver-java Authors
```

Comment:
```suggestion
 * Copyright 2022 The caver-java Authors
```

Reply:
Updated :) and fixed broken tests in RPCTests.